web/src/pages/onboarding.tsx

```diff
@@ -255,8 +259,8 @@ const OnboardingPage = (props: Props): ReactElement => {
 
     const isRemoteSellerWorker =
       newProfileData.businessPersona === "FOREIGN" &&
-      (newProfileData.foreignBusinessType === "REMOTE_SELLER" ||
-        newProfileData.foreignBusinessType === "REMOTE_WORKER");
+      (determineForeignBusinessType(newProfileData.foreignBusinessTypeIds) === "REMOTE_SELLER" ||
+        determineForeignBusinessType(newProfileData.foreignBusinessTypeIds) === "REMOTE_WORKER");
```

Comment:
[pebble] can we calculate these values in a variable and then use that here? This is just kindof long now.